src/hooks/triggers/useFetchConnections.ts

```diff
@@ -1,44 +1,35 @@
-// useFetchConnections.js
 import { useEffect, useState } from "react";
 
-import { ConnectionService } from "@services";
 import { extraTriggerTypes } from "@src/constants";
 import { SelectOption } from "@src/interfaces/components";
+import { useCacheStore } from "@src/store/cache/useCacheStore";
 
```

Comment:
You can remove this file. It is only to fetch and refresh the connections for triggers.
Use in trigger the regular cache store and format the connection also there.

Reply:
refactored